packages/x-components/src/components/base-dropdown.vue

```diff
@@ -31,22 +38,23 @@
         @keydown.end="highlightLastItem"
         @keydown.esc="close"
         @keydown.home="highlightFirstItem"
-        :aria-expanded="isOpen.toString()"
+        id="base-dropdown-listbox"
         class="x-dropdown__items-list"
         role="listbox"
-        tabIndex="0"
+        tabindex="0"
       >
         <li v-for="(item, index) in items" :key="item.id || item" class="x-dropdown__list-item">
           <button
             ref="itemButtons"
             @click="emitSelectedItemChanged(item)"
+            id="`dropdown-option-${value}`"
             :aria-selected="(index === highlightedItemIndex).toString()"
             :aria-current="(item === value).toString()"
             :class="itemsCSSClasses[index]"
             class="x-dropdown__item"
             data-test="dropdown-item"
             role="option"
-            tabindex="-1"
+            tabindex="0"
```

Comment:
Because this is a button, it is automatically focusable in the document order. `tabindex` should not be needed.